src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetsContainer/index.spec.tsx

```diff
@@ -128,8 +129,8 @@ describe('<DashboardVideoLiveWidgetContainer />', () => {
       </ResponsiveContext.Provider>,
     );
 
-    expect(container.firstChild).not.toBeNull();
-    const parentContainer = container.firstChild;
+    expect(container!.firstChild).not.toBeNull();
+    const parentContainer = container!.firstChild;
```

Comment:
:( can't we get rid of those asserts ?

Reply:
We could but I don't think we should any time on that since it is only in tests and at worst a test will raise an exception and so fails and that's pretty much what we want.